MatricesForHomalg/gap/Tools.gi

```diff
@@ -4553,17 +4553,21 @@ InstallMethod( Random,
     
 end );
 
+if IsPackageMarkedForLoading( "utils", ">= 0.54" ) then
+
 ##
 InstallMethod( Random,
         "for a homalg ring",
         [ IsHomalgRing ],
         
   function( R )
     
-    return Random( R, RandomCombination( [ 0 .. 10 ], Random( [ 1 .. 11 ] ) ) );
+    return Random( R, ValueGlobal( "RandomCombination" )( [ 0 .. 10 ], Random( [ 1 .. 11 ] ) ) );
```

Comment:
I think the `ValueGlobal` hack is not necessary because the whole block is wrapped in `IsPackageMarkedForLoading( "utils", ">= 0.54" )` anyway, or am I missing something?